README.md

```diff
@@ -1 +1,47 @@
+|----------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `LogzioToken (Required)`                       | Token for shipping metrics to your Logz.io account. Find it under Settings > Manage accounts. [_How do I look up my Metrics account token?_](/user-guide/accounts/finding-your-metrics-account-token/)   |
+| `LogzioListener (Required)`                    | Your logzio listener url (for metrics).                                                                                                                                                                  |
+ | `LambdaName (Optional)`                        | Name for the current lambda function. Default: x509-Certificate-Metrics-Lambda                                                                                                                           |
+| `CertificateURL (Required)`                    | The URL to collect x509 certificate metrics from, including port. i.e: https://app.logz.io:443                                                                                                           |
+| `LambdaTimeout (Optional)`                     | The amount of time that Lambda allows a function to run before stopping it. Minimum value is 1 second and Maximum value is 900 seconds. We recommend to start with 300 seconds (5 minutes). Default: 300 |
+| `CloudWatchEventScheduleExpression (Optional)` | The scheduling expression that determines when and how often the Lambda function runs. We recommend to start with 10 hour rate. Default: rate(10 hours)                                                  |
+| `LOGZIO_LOGS_TOKEN`                          | Your Logz.io log shipping [token](https://app.logz.io/#/dashboard/settings/manage-tokens/data-shipping).                                                                                                                                                                                                                                    | Required         |
+| `LOGZIO_LISTENER`                            | Your  Logz.io listener address, with port 8070 (http) or 8071 (https). For example: `https://listener.logz.io:8071`                                                                                                                                                                                                                         | Required         |
+| `LOGS_EXT_LOG_LEVEL`                         | Log level of the extension. Can be set to one of the following: `debug`, `info`, `warn`, `error`, `fatal`, `panic`.                                                                                                                                                                                                                         | Default: `info`  |
```

Comment:
The log extension parameters shouldnt be public as it for us to monitor the success of the lambda